### ISSUE TITLE
getMessage() method returns undefined in the notification handler

### ISSUE THREAD

**Issue by varungupta85** [react-native-community/push-notification-ios#169]
# Bug report

## Summary

`getMessage()` method returns `undefined` in the notification handler in the app for both local and remote notifications.

## Environment info

`react-native info` output:

```bash
System:
    OS: macOS 10.15.5
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 671.89 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.1.0 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.8.4 - /Users/varungupta/.rvm/rubies/ruby-2.4.1/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 21, 23, 24, 25, 26, 27, 28, 29
      Build Tools: 23.0.1, 25.0.0, 26.0.1, 26.0.2, 27.0.1, 27.0.3, 28.0.3, 29.0.2
      System Images: android-26 | Google APIs Intel x86 Atom, android-26 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-29 | Google Play Intel x86 Atom, android-30 | Google Play Intel x86 Atom
      Android NDK: 15.2.4203891
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6626763
    Xcode: 11.2.1/11B500 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_144 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 1.4.0

## Steps to reproduce

1. Run the example app
2. Tap on `Schedule fake local notification`
3. Tap on the notification that appears. You will see the following image

![image](https://user-images.githubusercontent.com/7475133/90018976-663d8100-dccb-11ea-9f50-ce669fec506a.png)

Describe what you expected to happen:

1. Alert message should be `Test Local Notification`

## Reproducible sample code

Use the example project as is without any change.

**Comment by varungupta85** [comment]
cc @Naturalclar This is a bread and butter use-case for this library that is broken at the moment. Please advice.

**Comment by nomi9995** [comment]
@varungupta85 i have fixed. you can check my pull request or https://stackoverflow.com/a/63477319/8079868 answer

**Comment by Naturalclar** [comment]
I've released version 1.4.1 with the fix. Thanks @nomi9995 !

**Comment by varungupta85** [comment]
@nomi9995 Great! Thanks so much! My app release was on hold due to this.
